process_manager/main.py

```diff
@@ -0,0 +1,48 @@
+from __init__ import ProcessManager
+import datetime
+
+from bin import SchedulerTask
+from scheduler import Scheduler
+
+if __name__ == "__main__":
+    
+    # TODO: This is going to be moved to a config file
```

Comment:
Ah. Okay, now I see why you used `str` instead of `Enum`, but I think that we can just do an `Enum` lookup when we parse the config file:

```
class BinType(str, Enum):
    BIANNUAL_BINS = 'biannual_bins'
   ...

BinType('biannual_bins')
# returns <BinType.BIANNUAL_BINS: 'biannual_bins'>
```
Then we would get a `ValueError` automatically if the config file was wrong.

Same with the parameter values inside the bins: this will ensure to at least a greater degree that we don't have incorrect entries in the `dict`.